readthedocs/projects/views/private.py

```diff
@@ -1096,7 +1100,6 @@ def get_context_data(self, **kwargs):
         context.update({
             'top_viewed_pages': top_viewed_pages,
             'page_data': page_data,
-            'enabled': self._is_enabled(project),
```

Comment:
Won't this fail if we don't pass `top_viewed_pages`? https://github.com/readthedocs/readthedocs.org/blob/0ff60a3cc9ed526d7065864c57d872999b0e4dc3/readthedocs/templates/projects/project_traffic_analytics.html#L21

Maybe that HTML block needs to check for `if enabled`, don't you think?

Reply:
When templates find null data, they just ignore it, in this case it goes to the `empty` block.